### PR TITLE
Add a test for VB line commit

### PR DESCRIPTION
**Pull request by rchande**
Tag @dotnet/roslyn-ide for review

### CODE REVIEW

**Comment by rchande** [comment]
Something about AddFile(open: true) doesn't actually open the file, hence the additional call to OpenFile. Will figure out what's going on at some point, but I can't actually debug the inproc stuff on my machine.

File: src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs [hunk 1]
```diff
@@ -0,0 +1,135 @@
+﻿
```
Comment (dpoeschl) [inline src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs]: Copyright

File: src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    public class BasicLineCommit : AbstractEditorTest
+    {
+        protected override string LanguageName => LanguageNames.VisualBasic;
+
+        public BasicLineCommit(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory, nameof(BasicLineCommit))
+        {
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
```
Comment (dpoeschl) [inline src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs]: There's a `Traits.Features.LineCommit` that you could use on all of these.

File: src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        void CommitOnFocusLost()
+        {
+            VisualStudio.Editor.SetText(@"Module M
+    Sub M()
+    End Sub
+End Module");
+
+            VisualStudio.Editor.PlaceCaret("End Sub", charsOffset:- 1);
+            VisualStudio.Editor.SendKeys(" ");
+            VisualStudio.SolutionExplorer.AddFile(new Microsoft.VisualStudio.IntegrationTest.Utilities.Common.ProjectUtils.Project(ProjectName), "TestZ.vb", open: true); // Cause focus lost
```
Comment (dpoeschl) [inline src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs]: I've been using an alias of just "ProjectUtils" in my tests to get around this craziness.

File: src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+        void CommitOnFocusLost()
+        {
+            VisualStudio.Editor.SetText(@"Module M
+    Sub M()
+    End Sub
+End Module");
+
+            VisualStudio.Editor.PlaceCaret("End Sub", charsOffset:- 1);
+            VisualStudio.Editor.SendKeys(" ");
+            VisualStudio.SolutionExplorer.AddFile(new ProjName(ProjectName), "TestZ.vb", open: true); // Cause focus lost
```
Comment (jasonmalinowski) [inline src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs]: Can you file a bug and mention it here so it's clear what is happening?

File: src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+End Module");
+
+            VisualStudio.Editor.PlaceCaret("(", charsOffset: 1);
+            VisualStudio.Editor.SendKeys("x   as   integer", VirtualKey.Tab);
+            VisualStudio.ExecuteCommand("File.SaveSelectedItems");
+            VisualStudio.Editor.Verify.TextContains(@"Sub Main(x As Integer)");
+            VisualStudio.ExecuteCommand(WellKnownCommandNames.Edit_Undo);
+            VisualStudio.Editor.Verify.TextContains(@"Sub Main(x   As   Integer)");
+            VisualStudio.Editor.Verify.CaretPosition(18);
+            VisualStudio.Editor.Verify.CaretPosition(16);
```
Comment (jasonmalinowski) [inline src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs]: What does Verify.CaretPosition doing that you can do it twice like this?? Can you add a comment to explain?

File: src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+        [Fact, Trait(Traits.Feature, Traits.Features.LineCommit)]
+        void CommitOnFocusLost()
+        {
+            VisualStudio.Editor.SetText(@"Module M
+    Sub M()
+    End Sub
+End Module");
+
+            VisualStudio.Editor.PlaceCaret("End Sub", charsOffset:- 1);
```
Comment (jasonmalinowski) [inline src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs]: Strange minus sign is strange.

File: src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs (hunk 5 above)
Comment (rchande) [inline src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicLineCommit.cs]: Unintentional, will remove the extra one.

**Comment by rchande** [comment]
Thanks @jasonmalinowski. Feedback incorporated.

**Comment by rchande** [comment]
retest windows_debug_vs-integration_prtest

**Comment by rchande** [comment]
retest windows_release_vs-integration_prtest

**Pull request merged**